packages/botonic-cli/src/botonic-api-service.ts

```diff
@@ -129,39 +129,56 @@ export class BotonicAPIService {
     if (pathExists(pathToCredentials)) unlinkSync(pathToCredentials)
   }
 
-  async api(
-    path: string,
-    body: any = null,
-    method: Method = 'get',
-    headers: any = null,
-    params: any = null
-  ): Promise<any> {
-    let b = 0
-    try {
-      return await axios({
-        method: method,
-        url: this.baseApiUrl + path,
+  async apiPost({
+    path,
```

Comment:
version as a parameter, by default v1

Reply:
Changed in the next [PR#2894](https://github.com/hubtype/botonic/pull/2894)